pdal/Geometry.cpp

```diff
@@ -67,21 +67,35 @@ Geometry::Geometry(const std::string& wkt_or_json, SpatialReference ref)
 
 
 Geometry::Geometry(const Geometry& input) : m_geom(input.m_geom->clone())
-{}
+{
+    if (input.m_geom)
+        m_geom.reset(input.m_geom->clone());
+}
```

Comment:
@hobu I think you have forgotten to remove the `m_geom` init through the initializer list - it is duplicated now (and thus it would still segfault)

Reply:
cleanup in #4020 ﻿